src/app/blogs/[id]/page.tsx

```diff
@@ -26,16 +26,24 @@ const post = {
 // TODO dummy page (Needs properly formatted page that accepts WYSIWYG)
 const BlogPost = () => {
   return (
-    <div className="mx-auto max-w-4xl p-4">
+    <div className="mx-auto min-h-screen max-w-4xl p-4">
```

Comment:
The footer was coming halfway up the page, let me know if there is a better way to get the footer to stick to the bottom 🤓 
![Screenshot 2024-07-16 at 9 39 40 AM](https://github.com/user-attachments/assets/d8f0b62e-9445-4271-a2c0-ddc05ac6a751)

Reply:
I think this has to be done in the template instead. Otherwise we will be doing this on every pages that do not have enough content to fit whole height.

Reply:
As @20was said you can try to maintain proper spacing with margin or padding in Navbar and Footer so there will be uniformity.

Reply:
Makes sense now that I have implemented this using global css and <footer> tag 👍